### ISSUE TITLE
Multiple @fieldTypes in JSON API

### ISSUE THREAD

**Issue by micky008** [orientechnologies/orientdb#9793]
## Expected behavior  
have an array of @fieldTypes 
```json
 {
            "@type": "d",
            "@rid": "#46:0",
            "@version": 5,
            "@class": "User",
            "name": "mich",
            "@fieldTypes": ["name=w","out_sold=g", "out_buy=g" ],
            "out_sold": [
                "#82:0"
            ],
            "out_buy": [
                "#70:0"
            ]
        }
```

the w type is for string (w for example)

## Actual behavior  
I have 1 @fieldTypes per Edge
(result in postman)
```json
{
            "@type": "d",
            "@rid": "#46:0",
            "@version": 5,
            "@class": "User",
            "name": "mich",
            "@fieldTypes": "out_sold=g",
            "out_sold": [
                "#82:0"
            ],
            "@fieldTypes": "out_buy=g",
            "out_buy": [
                "#70:0"
            ]
 }
```
result in Angular
```json
 {
            "@type": "d",
            "@rid": "#46:0",
            "@version": 5,
            "@class": "User",
            "name": "mich",
            "out_sold": [
                "#82:0"
            ],
            "@fieldTypes": "out_buy=g",
            "out_buy": [
                "#70:0"
            ]
        }
```
when a read the response, I've the only last @fieldTypes.

## Steps to reproduce  
CREATE CLASS User EXTENDS V
CREATE CLASS Vehicle EXTENDS V
CREATE CLASS buy EXTENDS E
CREATE CLASS sold EXTENDS E

insert into User set name = 'mich'
insert into Vehicle set name = 'nissan'
insert into Vehicle set name = 'GM'

CREATE EDGE buy FROM #idUserMich TO #idVehicleGM
CREATE EDGE sold FROM #idUserMich TO #idVehicleNissan

and in postman  (with basic auth root/root) : 
http://localhost:2480/connect/githubIssue 
http://localhost:2480/query/githubIssue/sql/select from V
```json
 {
            "@type": "d",
            "@rid": "#46:0",
            "@version": 5,
            "@class": "User",
            "name": "mich",
            "@fieldTypes": "out_sold=g",
            "out_sold": [
                "#82:0"
            ],
            "@fieldTypes": "out_buy=g",
            "out_buy": [
                "#70:0"
            ]
        }

```

**Comment by tglman** [comment]
Hi,

This should be fixed in 3.2.10.

Regards

**Comment by micky008** [comment]
it's ok. 
it's an array now.

the result:
```json
 "result": [
        {
            "@type": "d",
            "@rid": "#46:0",
            "@version": 5,
            "@class": "User",
            "name": "mich",
            "out_sold": [
                "#82:0"
            ],
            "out_buy": [
                "#70:0"
            ],
            "@fieldTypes": "out_sold=g,out_buy=g"
        },
        {
            "@type": "d",
            "@rid": "#58:0",
            "@version": 4,
            "@class": "Vehicle",
            "@fieldTypes": "in_buy=g",
            "name": "nissan",
            "in_sold": [
                "#82:0"
            ],
            "in_buy": [],
            "@fieldTypes": "in_sold=g,in_buy=g"
        },
        {
            "@type": "d",
            "@rid": "#59:0",
            "@version": 2,
            "@class": "Vehicle",
            "name": "GM",
            "in_buy": [
                "#70:0"
            ],
            "@fieldTypes": "in_buy=g"
        }
    ],
```
